hw/ip/spi_device/doc/_index.md

```diff
@@ -49,6 +69,38 @@ The SW can receive TPM commands with payload (address and data) and respond to t
 The submodule provides the command, address, write, and read FIFOs for the SW to communicate with the TPM host system.
 The submodule also supports the SW by managing a certain set of the FIFO registers and returning the read request by HW quickly.
 
+In Flash mode, SPI Device HWIP behaves as a Serial Flash device by recognizing SPI Flash commands and processing those commands by HW.
+The commands processed by HW are Read Status (1, 2, 3), Read JEDEC ID, Read SFDP, EN4B/ EX4B, and read commands with aids of SW.
+The IP supports Normal Read, Fast Read, Fast Read Dual Output, Fast Read Quad Output.
+This version of IP does not support Dual IO, Quad IO, QPI commands.
+
+In Passthrough mode, SPI Device receives SPI transactions from a host system and forwards the transactions to a downstream flash device.
+SW may filter prohibited commands by configuring 256-bit {{< regref "FILTER" >}} CSR.
+The IP cancels ongoing transaction if the received opcode matches to the filter CSR by de-asserting CSb and gating SCK  to the downstream flash device.
```

Comment:
extra space after SCK

Reply:
Blame my keyboard. It easily adds additional space ... I probably switch the switch .